src/Client/Web/Bit.Client.Web.BlazorUI.Tests/Modal/BitModalTests.cs

```diff
@@ -254,6 +254,67 @@ public void BitModalOnDismissShouldWorkCorrect(Visual visual)
             Assert.AreEqual(1, com.Instance.CurrentCount);
         }
 
+        [DataTestMethod,
+            DataRow(Visual.Fluent, BitModalPosition.Center),
+            DataRow(Visual.Fluent, BitModalPosition.TopLeft),
+            DataRow(Visual.Fluent, BitModalPosition.TopCenter),
+            DataRow(Visual.Fluent, BitModalPosition.TopRight),
+            DataRow(Visual.Fluent, BitModalPosition.CenterLeft),
+            DataRow(Visual.Fluent, BitModalPosition.CenterRight),
+            DataRow(Visual.Fluent, BitModalPosition.BottomLeft),
+            DataRow(Visual.Fluent, BitModalPosition.BottomCenter),
+            DataRow(Visual.Fluent, BitModalPosition.BottomRight),
+
+            DataRow(Visual.Cupertino, BitModalPosition.Center),
+            DataRow(Visual.Cupertino, BitModalPosition.TopLeft),
+            DataRow(Visual.Cupertino, BitModalPosition.TopCenter),
+            DataRow(Visual.Cupertino, BitModalPosition.TopRight),
+            DataRow(Visual.Cupertino, BitModalPosition.CenterLeft),
+            DataRow(Visual.Cupertino, BitModalPosition.CenterRight),
+            DataRow(Visual.Cupertino, BitModalPosition.BottomLeft),
+            DataRow(Visual.Cupertino, BitModalPosition.BottomCenter),
+            DataRow(Visual.Cupertino, BitModalPosition.BottomRight),
+
+            DataRow(Visual.Material, BitModalPosition.Center),
+            DataRow(Visual.Material, BitModalPosition.TopLeft),
+            DataRow(Visual.Material, BitModalPosition.TopCenter),
+            DataRow(Visual.Material, BitModalPosition.TopRight),
+            DataRow(Visual.Material, BitModalPosition.CenterLeft),
+            DataRow(Visual.Material, BitModalPosition.CenterRight),
+            DataRow(Visual.Material, BitModalPosition.BottomLeft),
+            DataRow(Visual.Material, BitModalPosition.BottomCenter),
+            DataRow(Visual.Material, BitModalPosition.BottomRight)
```

Comment:
Why have you added visual as a parameter here?
Is there any problem with removing it?

Reply:
I think this is a parameter that we must follow in all tests (regardless of what property it is written for...) so that our tests work properly for all three different designs.
What do you think? @msynk @maryamhdr

Reply:
I'd recommend keep using it. Because it make sure components behavior remains the same no matter what visual is being used

Reply:
let's keep them for now 😉